Source/WebKit/UIProcess/Cocoa/UserMediaCaptureManagerProxy.h

```diff
@@ -70,8 +70,10 @@ class UserMediaCaptureManagerProxy : public IPC::MessageReceiver {
         virtual void addMessageReceiver(IPC::ReceiverName, IPC::MessageReceiver&) = 0;
         virtual void removeMessageReceiver(IPC::ReceiverName) = 0;
         virtual IPC::Connection& connection() = 0;
+        virtual Ref<IPC::Connection> protectedConnection() = 0;
```

Comment:
This doesn't technically need to be virtual and overriden in subclasses, we could just do:
```
Ref<IPC::Connection> protectedConnection() { return connection(); }
```
here.

Reply:
Fixed!